apartmenttherapy/order-service
# Elaine/remove lat lng

#### What does this PR do & why?
We only needed the address lat/lng to create a unique id. But that means we have to interface with a 3rd party api, which is potentially costly and inconvenient. We should be able to use the `street1` + `street2` + `addressee` combo to create a unique id.

#### How can this be manually tested?
- Tests should run.
- Create an address (e.g. through an order). Verify the address's id in the db is `street1__street2__addressee`.

#### Any questions or concerns about this change?
*(Any significant changes you're introducing? What side effects might this have? What other parts of the app could this affect?)*


#### Deployment Notes
*(e.g. Does this require a migration or special infrastructure change? Clearing the cache? Third party service?)*
This branch has been deployed to dev for testing.

#### Link to story card and other documentation


#### GIF


#### Have you remembered to...?
